niri-config/src/lib.rs

```diff
@@ -989,6 +1010,59 @@ impl From<niri_ipc::Action> for Action {
     }
 }
 
+#[derive(knuffel::Decode, Debug, PartialEq, Clone)]
```

Comment:
I guess this Decode derive is not needed? And you can add `Eq`.
```suggestion
#[derive(Debug, PartialEq, Eq, Clone)]
```

Reply:
Indeed, the Decode is not needed, nor the `#[knuffel(argument)]` stuff in the following lines. Thanks!